include/wx/translation.h

```diff
@@ -151,16 +151,24 @@ class WXDLLIMPEXP_BASE wxTranslations
     // get languages available for this app
     wxArrayString GetAvailableTranslations(const wxString& domain) const;
 
-    // find best translation language for given domain
+    // find best available translation language for given domain
+    wxString GetBestAvailableTranslation(const wxString& domain);
+
     wxString GetBestTranslation(const wxString& domain, wxLanguage msgIdLanguage);
     wxString GetBestTranslation(const wxString& domain,
                                 const wxString& msgIdLanguage = wxASCII_STR("en"));
 
+    // add catalog for the given domain returning true if it could be found by
+    // wxTranslationsLoader
+    bool AddAvailableCatalog(const wxString& domain);
```

Comment:
Yes, as you can probably guess, I spent a long time agonizing over the name, but I couldn't find anything better, so I just went ahead with "Available" because it was already used for `GetAvailableTranslations()` and it seemed better to avoid introducing yet another term.